infra/ipfs-proxy/src/app.js

```diff
@@ -39,6 +42,94 @@ function isValidVideo(buffer) {
   return file && validVideoTypes.includes(file.mime)
 }
 
+function getMultiHash(buffer) {
+  const unixFs = new Unixfs("file", buffer)
+  return new Promise((resolve, reject) => {
+    dagPB.DAGNode.create(unixFs.marshal(), 
```

Comment:
Can't find this in the docs or source for `js-ipld-dag-pb`. Older version?  This PR tested?